strlib.inc

```diff
@@ -523,6 +537,68 @@ stock bool:isequal(const str1[], const str2[], bool:ignorecase = false) {
     return !strcmp(str1, str2, ignorecase);
 }
 
+stock strdistance(const str1[], const str2[], bool:ignorecase = false) {
+	// If they are equal
+	if(isequal(str1, str2, ignorecase))
+		return 0;
+
+	// Data
+	static d[128][128];
```

Comment:
Perhaps just name the variable `data` and you won't need the comment explaining what the variable is.

Reply:
I mean, yeh.